app/src/main/java/bluedot/com/au/bluedotbrazeintegrationapp/BluedotGeoTriggerReceiver.kt

```diff
@@ -2,47 +2,46 @@ package bluedot.com.au.bluedotbrazeintegrationapp
 
 import android.content.Context
 import au.com.bluedot.point.net.engine.GeoTriggeringEventReceiver
-import au.com.bluedot.point.net.engine.ZoneInfo
 import au.com.bluedot.point.net.engine.event.GeoTriggerEvent
 import com.braze.Braze
 import com.braze.models.outgoing.BrazeProperties
-import java.util.*
+import java.util.Date
 
 class BluedotGeoTriggerReceiver: GeoTriggeringEventReceiver() {
     private val customEventEntry = "bluedot_entry"
     private val customEventExit = "bluedot_exit"
 
-    override fun onZoneEntryEvent(geoTriggerEvent: GeoTriggerEvent, context: Context) {
-        println("Zone ${geoTriggerEvent.zoneInfo.name}, fence ${geoTriggerEvent.entryEvent()?.fenceName} entered at: ${Date()}")
+    override fun onZoneEntryEvent(entryEvent: GeoTriggerEvent, context: Context) {
+        println("Zone ${entryEvent.zoneInfo.name}, fence ${entryEvent.entryEvent()?.fenceName} entered at: ${Date()}")
 
         val eventProperties = BrazeProperties()
-        eventProperties.addProperty("zone_id", geoTriggerEvent.zoneInfo.id)
-        eventProperties.addProperty("zone_name", geoTriggerEvent.zoneInfo.name)
+        eventProperties.addProperty("zone_id", entryEvent.zoneInfo.id.toString())
+        eventProperties.addProperty("zone_name", entryEvent.zoneInfo.name)
         eventProperties.addProperty("latitude",
-            geoTriggerEvent.entryEvent()?.locations?.get(0)?.latitude
+            entryEvent.entryEvent()?.locations?.get(0)?.latitude
         )
-        eventProperties.addProperty("longitude", geoTriggerEvent.entryEvent()?.locations?.get(0)?.longitude)
-        eventProperties.addProperty("fence_id", geoTriggerEvent.entryEvent()?.fenceId)
-        eventProperties.addProperty("fence_name", geoTriggerEvent.entryEvent()?.fenceName)
+        eventProperties.addProperty("longitude", entryEvent.entryEvent()?.locations?.get(0)?.longitude)
+        eventProperties.addProperty("fence_id", entryEvent.entryEvent()?.fenceId.toString())
+        eventProperties.addProperty("fence_name", entryEvent.entryEvent()?.fenceName)
 
-        geoTriggerEvent.zoneInfo.customData?.forEach { data ->
+        entryEvent.zoneInfo.customData.forEach { data ->
```

Comment:
[Q/I] Is customData an optional in Android SDK:
https://android-docs.bluedot.io/-bluedot%20-s-d-k%20-docs/au.com.bluedot.point.net.engine/-zone-info/index.html

Reply:
customData is not optional in NotificationZoneInfo: https://android-docs.bluedot.io/-bluedot%20-s-d-k%20-docs/au.com.bluedot.point.net.engine.event/-notification-zone-info/index.html
Starting from SDK16 `GeoTriggerEvent` returns `NotificationZoneInfo` as `zoneInfo`